src/core/qgsdataprovider.h

```diff
@@ -74,7 +91,10 @@ class CORE_EXPORT QgsDataProvider : public QObject
       CustomData   = 3000          //!< Custom properties for 3rd party providers or very provider-specific properties which are not expected to be of interest for other providers can be added starting from this value up.
     };
 
-    QgsDataProvider( QString const &uri = "" )
+    /**
+     * Create a new dataprovider with the specified in the \a uri.
```

Comment:
Hi @m-kuhn, is something missing in this sentence?

Reply:
Should we just drop the "in the"?

Reply:
...or "with the data source specification defined in the uri"?

Reply:
Whatever you think adds the most information for a reader :+1: